app/assets/javascripts/discourse/app/components/modal/fullscreen-code.gjs

```diff
@@ -25,4 +28,21 @@ export default class FullscreenCode extends Component {
     });
     this.codeBlockButtons.attachToGeneric(modalElement);
   }
+
+  <template>
+    <DModal
+      @title={{i18n "copy_codeblock.view_code"}}
+      @closeModal={{this.closeModal}}
+      {{didInsert this.applyCodeblockButtons}}
+      class="fullscreen-code-modal -max"
+    >
+      <:body>
+        <pre>
+      <code
+            class={{@model.codeClasses}}
+          >{{@model.code}}</code>
+    </pre>
+      </:body>
```

Comment:
(this is a now-fixed behavior of prettier after I reported it messing with whitespace in `pre` blocks 😛  looks bad but makes sense 😉 we can tweak it manually here though)

```suggestion
      <:body>
        <pre>
          <code
            class={{@model.codeClasses}}
          >{{@model.code}}</code>
        </pre>
      </:body>
```